static/app/views/ddm/customMetricsEventData.tsx

```diff
@@ -0,0 +1,206 @@
+                  ? {
+                      key: 'tags',
+                      subject: t('Tags'),
+                      value: (
+                        <Tags tags={item.tags} organization={organization} mri={mri} />
+                      ),
+                    }
+                  : null,
+              ].filter((row): row is Exclude<typeof row, null> => Boolean(row))}
+            />
+          </Fragment>
+        );
+      })}
+    </EventDataSection>
+  );
+}
+
+function MetricStats({mri, item}: {item: MetricsSummaryItem; mri: MRI}) {
+  const parsedMRI = parseMRI(mri);
+  const unit = parsedMRI?.unit ?? 'none';
+  const type = parsedMRI?.type ?? 'c';
+
+  const typeLine = t(`Type: %s`, getReadableMetricType(type));
+  // We use formatMetricUsingUnit with unit 'none' to ensure uniform number formatting
+  const countLine = t(`Count: %s`, formatMetricUsingUnit(item.count, 'none'));
+
+  const baseStats = (
+    <Fragment>
+      {typeLine}
+      <br />
+      {countLine}
+    </Fragment>
+  );
+
+  // For counters the other stats offer little value, so we only show type and count
+  if (type === 'c' || !item.count) {
+    return <pre>{baseStats}</pre>;
+  }
+
+  // If there is only one value, min, max, avg and sum are all the same
+  if (item.count <= 1) {
+    return (
+      <pre>
+        {baseStats}
+        <br />
+        {t('Value: %s', formatMetricUsingUnit(item.sum, unit))}
+      </pre>
+    );
+  }
+
+  return (
+    <pre>
+      {baseStats}
+      <br />
+      {t('Sum: %s', formatMetricUsingUnit(item.sum, unit))}
+      <br />
+      {t('Min: %s', formatMetricUsingUnit(item.min, unit))}
+      <br />
+      {t('Max: %s', formatMetricUsingUnit(item.max, unit))}
+      <br />
+      {t(
+        'Avg: %s',
+        formatMetricUsingUnit(item.sum && item.count && item.sum / item.count, unit)
+      )}
+    </pre>
+  );
+}
+
+function Tags({
+  tags,
+  organization,
+  mri,
+}: {
+  mri: MRI;
+  organization: Organization;
+  tags: Record<string, string>;
+}) {
+  const [showingAll, setShowingAll] = useState(false);
+
+  const renderedTags = Object.entries(tags).slice(0, showingAll ? undefined : 5);
+  const renderText = showingAll ? t('Show less') : t('Show more') + '...';
+
+  return (
+    <StyledPills>
+      {renderedTags.map(([tagKey, tagValue]) => (
+        <Pill key={tagKey} name={tagKey}>
+          <Link
+            to={getDdmUrl(organization.slug, {
+              widgets: [
+                {
+                  mri,
+                  displayType: MetricDisplayType.LINE,
+                  op: getDefaultMetricOp(mri),
+                  query: `${tagKey}:"${tagValue}"`,
+                },
+              ],
+            })}
+          >
+            <StyledTextOverflow>{tagValue}</StyledTextOverflow>
+          </Link>
+        </Pill>
+      ))}
+      {Object.entries(tags).length > 5 && (
+        <ShowMore onClick={() => setShowingAll(prev => !prev)}>{renderText}</ShowMore>
+      )}
```

Comment:
Any chance we could prevent this jank?
https://github.com/getsentry/sentry/assets/9060071/71154655-3bf3-4819-a7a0-9984ae600eb5

Reply:
Hmm, I replicated the behaviour of the transaction tags to have a consistent UX.
What exactly is the issue you are referring to? That the toggle is inline with the tags?

Reply:
There's a layout shift - the entire right column jumps when you click Show More.

Reply:
ah, got it 👍 
I checked it a bit, we have the same problem with the other tags and I don't see an easy way to fix it without touching the base component for the Pills 🤔 
Will not fix it in this PR.